### PR TITLE
Enable Rook deployments (#884)

### PR DESCRIPTION
**Pull request by yeoldegrove**
This is part of https://github.com/osism/issues/issues/884

### CODE REVIEW

**Comment by yeoldegrove** [comment]
@berendt Any idea how to differentiate between `cephclient_install_type` ?

https://github.com/osism/testbed/pull/2274/files#diff-5c991384148c260c4cdcb2c128295f9093039429b05ec6c4c9ea894692c47647R9

File: Makefile [hunk 1]
```diff
@@ -81,7 +83,8 @@ bootstrap: setup create ## Bootstrap everything.
 	  -e manual_deploy=true \
 	  -e version_ceph=$(VERSION_CEPH) \
 	  -e version_manager=$(VERSION_MANAGER) \
-	  -e version_openstack=$(VERSION_OPENSTACK)
+	  -e version_openstack=$(VERSION_OPENSTACK) \
+	  -e ceph_stack=$(CEPH_STACK) \
```
Comment (berendt) [inline Makefile:87]: Remove trailing \

File: playbooks/deploy.yml [hunk 2]
```diff
@@ -31,6 +31,8 @@
     run_refstack: "{{ refstack | default(false) | bool }}"
     run_tempest: "{{ tempest | default(false) | bool }}"
 
+    ceph_stack: "{{ ceph_stack | default('ceph-ansible') }}"
```
Comment (berendt) [inline playbooks/deploy.yml:34]: We have to use a different variable name here. I will touch this first and rename everything in e.g. `_tempest`.

File: playbooks/vars/repositories.yml [hunk 3]
```diff
@@ -9,6 +9,10 @@ repositories:
     path: github.com/osism/ansible-collection-services
     repo: https://github.com/osism/ansible-collection-services
     branch: main
+  ansible-playbooks:
+    path: github.com/osism/ansible-playbooks
+    repo: https://github.com/osism/ansible-playbooks
+    branch: main
```
Comment (berendt) [inline playbooks/vars/repositories.yml:15]: Why is this required?

**Review by berendt — CHANGES_REQUESTED** [review]

File: environments/infrastructure/configuration.yml [hunk 4]
```diff
@@ -6,6 +6,7 @@ cephclient_mons:
   - 192.168.16.10
   - 192.168.16.11
   - 192.168.16.12
+cephclient_install_type: rook
```
[the remark this answers is not in the document]
Comment (yeoldegrove) [reply environments/infrastructure/configuration.yml:9]: Alright, we need to implement it later somehow.

File: playbooks/vars/repositories.yml (hunk 3 above)
Comment (yeoldegrove) [reply playbooks/vars/repositories.yml:15]: It is not needed actually. Is a relic from testing.

File: playbooks/deploy.yml (hunk 2 above)
Comment (berendt) [reply playbooks/deploy.yml:34]: Please rebase & use `_ceph_stack` inside the play.

Comment (yeoldegrove) [reply playbooks/deploy.yml]: @berendt Please have a look.
